components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/ParentalAgeController.java

```diff
@@ -181,7 +179,11 @@ public void writeJSON(Patient patient, JSONObject json, Collection<String> selec
         for (String property : getProperties()) {
             if (data.has(property)) {
                 int age = data.getInt(property);
-                result.put(property, age);
+                if (age == 0) {
+                    result.put(property, null);
```

Comment:
Is there a difference between age 0 and age == null? Don't both logically mean "no age is set", and if yes, isn't "null" a better indication?